packages/fe-fpm-writer/src/common/types.ts

```diff
@@ -13,6 +13,10 @@ export interface TabInfo {
  * Common properties for any custom element of the flexible programming model.
  */
 export interface CustomElement extends WriterConfig {
+    /**
+     * Name of the custom element fragment.
+     */
+    fragmentName?: string;
```

Comment:
should be moved to differnet interface, because not all custom extensions has fragment, for example custom action